packages/node_modules/@webex/internal-plugin-conversation/src/config.js

```diff
@@ -32,6 +32,14 @@ export default {
      * @type {Boolean}
      */
     keepEncryptedProperties: false,
-    decryptionFailureMessage: 'This message cannot be decrypted'
+    decryptionFailureMessage: 'This message cannot be decrypted',
+    /**
+     * update key activity constants
+     */
+    updateKeyActivity: {
+      PERSON: 'person',
+      CONVERSATION: 'conversation',
+      UPDATE_KEY: 'updateKey'
+    }
```

Comment:
If these are constants, they can be placed in a file relative to the plugin: `./src/constants.js` and imported this way. See [this file](https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/internal-plugin-devices/src/constants.js) for an example.

Reply:
will move into constant file.